community/modules/project/service-account/variables.tf

```diff
@@ -20,26 +20,25 @@ variable "billing_account_id" {
   default     = ""
 }
 
-variable "description" {
-  description = "Default description of the created service accounts (defaults to no description)."
+variable "deployment_name" {
+  description = "Name of the deployment (will be prepended to service account name)"
   type        = string
-  default     = ""
 }
 
-variable "descriptions" {
```

Comment:
I feel like the correct thing to do with variables that no longer exist would be to leave an unused variable in place that does a validation, which would fail for anything other than null. The error message would direct users to a good course of action to update their blueprint.

Reply:
Done